DuckDuckGo/TabViewController.swift

```diff
@@ -218,6 +228,25 @@ class TabViewController: UIViewController {
         shouldReloadOnError = true
     }
     
+    override func pressesBegan(_ presses: Set<UIPress>, with event: UIPressesEvent?) {
+        super.pressesBegan(presses, with: event)
+        
+        guard #available(iOS 13.4, *) else { return }
+        
+        if event?.modifierFlags.contains(.command) ?? false {
+            if event?.modifierFlags.contains(.shift) ?? false {
+                tapLinkDestination = .newTab
+            } else {
+                tapLinkDestination = .backgroundTab
+            }
+        }
+    }
+    
+    override func pressesEnded(_ presses: Set<UIPress>, with event: UIPressesEvent?) {
+        super.pressesEnded(presses, with: event)
+        tapLinkDestination = .currentTab
```

Comment:
Try this:
- Press and hold cmd.
- Press and release shift.
- While still holing cmd, tap on a link.

Expected: link opens in new tab.
Actual: link is just opened.